st2actions/st2actions/policies/concurrency.py

```diff
@@ -54,7 +54,7 @@ def _apply_before(self, target):
             LOG.debug('There are %s instances of %s in scheduled or running status. '
                       'Threshold of %s is not reached. Action execution will be scheduled.',
                       count, target.action, self._policy_ref)
-            status = action_constants.LIVEACTION_STATUS_SCHEDULED
+            status = action_constants.LIVEACTION_STATUS_REQUESTED
```

Comment:
At this point it is not scheduled, as it would have been previously. The scheduler will move it into scheduled state if the rest of its checks pass. Eventually priority, delay, etc.

Reply:
Can you point me to where the scheduler move it into scheduled state?  I don't seem to find the change in the PR.

Reply:
I didn't make the change. It's current functionality: https://github.com/StackStorm/st2/blob/87d991e5fc2cf383a25c89a6a563161bd86dd284/st2actions/st2actions/policies/concurrency.py#L57